pkg/chunked/storage_linux.go

```diff
@@ -146,40 +146,48 @@ func (c *chunkedDiffer) convertTarToZstdChunked(destDirectory string, payload *o
 func GetDiffer(ctx context.Context, store storage.Store, blobDigest digest.Digest, blobSize int64, annotations map[string]string, iss ImageSourceSeekable) (graphdriver.Differ, error) {
```

Comment:
The documentation of `GetDiffer` should refer to the special error type, and explain the semantics.